pkg/util/log/crash_reporting.go

```diff
@@ -66,6 +66,12 @@ var (
 	// startTime records when the process started so that crash reports can
 	// include the server's uptime as an extra tag.
 	startTime = timeutil.Now()
+
+	// ReportUnanonymized enables reporting of unanonymized data.
+	//
+	// This should not be used by anyone unwilling to share the whole cluster
+	// data with Cockroach Labs and various cloud services.
+	ReportUnanonymized = envutil.EnvOrDefaultBool("COCKROACH_REPORT_UNANONYMIZED", false)
```

Comment:
unanymized.. nonymized... anon.. ugh! this is a mouthful. How about "uncensored", "unredacted", "..._SENSITIVE_DETAILS`, etc...

Reply:
The first isn't correct, the second one isn't clear enough, I could do the third.

Reply:
I think the third is clear and describes the danger pretty well. Bonus points for being easier to say.

Reply:
Yup, done.